jetty-core/jetty-quic/jetty-quic-quiche/jetty-quic-quiche-foreign/src/main/java/org/eclipse/jetty/quic/quiche/foreign/ForeignQuicheConnection.java

```diff
@@ -44,6 +44,11 @@ public class ForeignQuicheConnection extends QuicheConnection
     private static final Logger LOG = LoggerFactory.getLogger(ForeignQuicheConnection.class);
     private static final SecureRandom SECURE_RANDOM = new SecureRandom();
 
+    static
+    {
+        quiche_h.versionCheck();
```

Comment:
Why is this static block in this class? Loading `quiche_h` would trigger static blocks, so feels like this block should be there.

Reply:
As we discussed, this has been cleaned up.